yb-voyager/src/query/queryissue/issues_ddl_test.go

```diff
@@ -393,6 +383,56 @@ EXECUTE FUNCTION public.check_sales_region();`)
 	assertErrorCorrectlyThrownForIssueForYBVersion(t, err, `"sales_region" is a partitioned table`, beforeRowTriggerOnPartitionTableIssue)
 }
 
+func testDatabaseOptions(t *testing.T) {
+	sqlsforPG15 := []string{
+		` CREATE DATABASE locale_example
+    WITH LOCALE = 'en_US.UTF-8'
+         TEMPLATE = template0;`,
+		`CREATE DATABASE locale_provider_example
+    WITH ICU_LOCALE = 'en_US'
+         LOCALE_PROVIDER = 'icu'
+         TEMPLATE = template0;`,
+		`CREATE DATABASE oid_example
+    WITH OID = 123456;`,
+		`CREATE DATABASE collation_version_example
+    WITH COLLATION_VERSION = '153.128';`,
+		`CREATE DATABASE strategy_example
+    WITH STRATEGY = 'wal_log';`,
+	}
+	sqlsForPG17 := []string{
+		`CREATE DATABASE icu_rules_example
+    WITH ICU_RULES = '&a < b < c';`,
+		`CREATE DATABASE builtin_locale_example
+    WITH BUILTIN_LOCALE = 'C';`,
+	}
+
+	for _, sql := range sqlsforPG15 {
+		ctx := context.Background()
+		conn, err := getConn()
+		assert.NoError(t, err)
+
+		defer conn.Close(context.Background())
+		_, err = conn.Exec(ctx, sql)
+		switch {
```

Comment:
Wait, why do we need this switch case? If it's 2.25, err will be empty, and `assertErrorCorrectlyThrownForIssueForYBVersion` will take care of it automatically, right?

Reply:
we need the switch case right, because firstly we are not marking this as fixed in 2.25 but the CREATE DATABASE works without error in the `2.25` so, we need to run this func `assertErrorCorrectlyThrownForIssueForYBVersion` with different expected and actual in different cases

Reply:
let's add a comment here as well to indicate that it is not actually supported in 2.25, but does not throw any error as such